sources/PPU/PPU.hpp

```diff
@@ -566,10 +54,12 @@ namespace ComSquare::PPU
 		Registers _registers{};
 		Renderer::IRenderer &_renderer;
 		//! @brief Backgrounds buffers
-		Background _backgrounds[8];
+		Background _backgrounds[4];
 		//! @brief Main Screen buffer
 		std::array<std::array<uint32_t, 1024>, 1024> _mainScreen;
+		std::array<std::array<unsigned char, 1024>, 1024> _mainScreenLevelMap;
 		//! @brief Sub Screen buffer
+		std::array<std::array<unsigned char, 1024>, 1024> _subScreenLevelMap;
```

Comment:
That's not really consistent, you should use uint8_t.

Reply:
fixed